lib/worker.js

```diff
@@ -39,6 +39,23 @@ Worker.prototype._getConfigUpdateAction = function() {
                 reject(new Error('Invalid message received: ' + JSON.stringify(message)));
             }
         });
+
+        if (cluster.isWorker) {
+            // If got a status update in a worker - forward it to the master
+            process.on('service_status', function(message) {
+                try {
+                    process.send({
+                        type: 'service_status',
+                        status: message
+                    });
+                } catch (e) {
```

Comment:
We need a `try-catch` here as the circular references in the message will throw an error.